README.md

```diff
@@ -1 +1,4 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+Hello Microverse in an H1 tag.
+
+Built With HTML5 & CSS3.
```

Comment:
- [ ] As the first reviewer mentioned,Please add a descriptive README file to your project by using [this template](https://github.com/microverseinc/readme-template) provide by microverse, and please customize it to your project because it is important to keep your project profession :smile: 
- [ ] And Please don't forget to add a license to your project and I recommend using the [MIT license](https://choosealicense.com/licenses/mit/), Set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). another resource is [any other license](https://choosealicense.com/licenses/) . :smile: